### ISSUE TITLE
algorithm error generating aria-level for caution headings

### ISSUE THREAD

**Issue by WBerndorfer** [mattgarrish/epub-revision#2567]
Seems, there is a coding error in the realization of heading levels for caution-notes on some W3C sites:
https://github.com/w3c/epub-specs/blob/181828b283ee5e20546746057076d970e78ce1ea/epub33/common/js/add-caution-hd.js#L16

Example:
https://www.w3.org/TR/epub-33/#sec-intro-relations
aria-heading with an extraordinary value 31 for aria-level:
&lt;div class="caution" id="caution-0">
&lt;div
  role="heading"
  aria-level="31"
  class="caution-title marker">
&lt;span>Caution&lt;/span>
&lt;/div>

Two digits concatenated together, but should be added mathematically.

**Comment by iherman** [comment]
This seems to be a [class-1](https://www.w3.org/2023/Process-20230612/#class-1) modification request. I also believe it is a valid Erratum, so we should mark it as such

@wareid @shiestyle @TzviyaSiegman @dauwhe
